### PR TITLE
Userflow in miro Iteration1

### PR DESCRIPTION
**Pull request by naila135**
Reolve #32

### CODE REVIEW

File: Specification/UserFlow/Iterations/Iteration_1.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This is first iteration of UserFlow for Sleeplog.<br>
```
Comment (sahlet-official) [inline Specification/UserFlow/Iterations/Iteration_1.md:1]: Well done!

File: Specification/UserFlow/Iterations/Iteration_1.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+Userflow was created based on:
```
Comment (sahlet-official) [inline Specification/UserFlow/Iterations/Iteration_1.md:2]: Change whole text to:

This is iteration 1 of UserFlow for Sleeplog.< br>
Userflow was created based on:
- the interview with the Product Owner;
- Product Owner requirements that were conducted from text dialog;
- [app functions] (./../AppFunctions.md);

Userflow was created in [Miro] (https://miro.com/) and can be seen [here] (https://miro.com/app/board/uXjVP9crbcw=/).

________________________

and remove spaces that I did in the links

image

File: Specification/UserFlow/Iterations/Iteration_1.md (hunk 1 above)
Comment (sahlet-official) [inline Specification/UserFlow/Iterations/Iteration_1.md:1]: Rename doc, please to

Iteration1.md

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: Specification/UserFlow/Iterations/Iteration_1.md (hunk 2 above)
Comment (naila135) [reply Specification/UserFlow/Iterations/Iteration_1.md]: image
It looks weird a little

File: Specification/UserFlow/Iterations/Iteration_1.md (hunk 1 above)
Comment (naila135) [reply Specification/UserFlow/Iterations/Iteration_1.md]: done

**Review by sahlet-official — COMMENTED** [review]
Rename PR please to

Userflow in miro Iteration1

**Review by sahlet-official — CHANGES_REQUESTED** [review]
Rename PR please to

Userflow in miro Iteration1

**Review by sahlet-official — APPROVED** [review]

**Pull request merged**